course_catalog/tasks.py

```diff
@@ -54,6 +65,13 @@ def get_edx_data(force_overwrite=False):
             break
 
         url = response.json()["next"]
+    edx_data["count"] = len(edx_data["results"])
+    raw_data_bucket = boto3.resource(
+        "s3",
+        aws_access_key_id=settings.OCW_CONTENT_ACCESS_KEY,
+        aws_secret_access_key=settings.OCW_CONTENT_SECRET_ACCESS_KEY,
+    ).Bucket(name=settings.OCW_CONTENT_BUCKET_NAME)
+    raw_data_bucket.put_object(Key="edx_courses.json", Body=json.dumps(edx_data))
```

Comment:
Can you add a unit test that mocks `put_object` and verifies that it is called with the correct arguments here (`Key="edx_courses.json", Body=json.dumps(edx_data)`)

Reply:
Added.